ctapipe/instrument/camera.py

```diff
@@ -203,6 +203,21 @@ def _calc_pixel_area(pix_x, pix_y, pix_type):
 
         return np.ones(pix_x.shape) * area
 
+    @lazyproperty
+    def all_pixels_kdtree(self):
+        """
+        Pre-calculated kdtree of all pixel centers inside camera
+
+
+        Returns
+        -------
+        kdtree
+
+        """
+
+        pixel_centers = np.stack([self.pix_x.to_value(u.m), self.pix_y.to_value(u.m)]).T
```

Comment:
`np.column_stack` gives directly what you want here without having to transpose.

Reply:
done in e43e781.